package.json

```diff
@@ -6,7 +6,12 @@
   "license": "MIT",
   "homepage": "https://github.com/giakki/uncss",
   "scripts": {
-    "test": "grunt test"
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "eslint": "eslint \"**/*.js\"",
+    "mocha": "mocha tests/*.js --reporter spec --slow 7500 --timeout 25000",
+    "lint": "npm run eslint",
+    "test": "npm run eslint && npm run mocha",
+    "travis": "nyc mocha tests/*.js --slow 7500 --timeout 25000"
```

Comment:
I had tried it before and indeed worked but I wasn't sure it's OK so I went with the safe solution of duplication :P